sponsorship_switzerland/models/account_invoice.py

```diff
@@ -93,3 +93,15 @@ def group_or_split_reconcile(self):
         """Reconcile given invoices with partner open payments.
         """
         return self._group_or_split_reconcile()
+
+    def action_invoice_open(self):
+        for invoice in self:
+            condition = [
+                ("transaction_id", "=", invoice.transaction_id),
+                ("state", "=", "open"),
+                ("id", "!=", invoice.id),
+            ]
+            if self.env[self._name].search(condition):
```

Comment:
This will be slighlty more performant and work the same (doing a count(*) query instead of fetching all data of records).
```suggestion
            if self.env[self._name].search_count(condition):
```